### ISSUE TITLE
Total memory re fails to validate applied memory in cmd-config feature

### ISSUE THREAD

**Issue by agajdosi** [minishift/minishift#2054]
Upstream issue for CDK JIRA https://issues.jboss.org/browse/CDK-256. The cmd-config.feature fails on Windows 10 slave with Hyper-V due to memory regex check:

```
$make integration MINISHIFT_BINARY=/c/cdk3.4alpha1/minishift GODOG_OPTS="-tags cmd-config"
--
[...]
Scenario: Checking that memory value was applied               # features\a.feature:214
When executing "minishift ssh -- cat /proc/meminfo" succeeds # testsuite.go:587 -> github.com/minishift/minishift/test/integration/testsuite.executingMinishiftCommandSucceedsOrFails
Then stdout should match "MemTotal:\s*3[4-5][0-9]{5} kB"     # testsuite.go:565 -> github.com/minishift/minishift/test/integration/testsuite.commandReturnShouldMatchRegex
Output did not match. Expected: 'MemTotal:\s*3[4-5][0-9]{5} kB', Actual: 'MemTotal:        3765156 kB
MemFree:         1216932 kB
MemAvailable:    2705220 kB
Buffers:           30672 kB
[...]
```

**Comment by LalatenduMohanty** [comment]
@agajdosi So it only fails in Windows 10, right? Wondering why we catch the issue now and why not in the past?

**Comment by agajdosi** [comment]
@LalatenduMohanty Interestingly the VM starts with `--memory 3500` so I am curious how in the JIRA we got 3700000kb.

**Comment by LalatenduMohanty** [comment]
Resolved via #2055
